static/includes/RootLevelDatasetShareWarning.md

```diff
@@ -0,0 +1,7 @@
+&NewLine;
+
+{{< hint type=tip title="Do Not Create Pool-Level Dataset Shares" >}}
```

Comment:
Wondering if this would be better as a yellow box than green, since it's more than just a tip.

```suggestion
{{< hint type=important title="Do Not Create Pool-Level Dataset Shares" >}}
```